chapters/7-relationships-between-SPDX-elements.md

```diff
@@ -76,6 +87,14 @@ Examples:
     Relationship: SPDXRef-DOCUMENT AMENDS DocumentRef-SPDXA:SPDXRef-DOCUMENT
 
     RelationshipComment: This current document is an amendment of the SPDXA document.
+    
+    Relationship: SPDXRef-CarolCompression DEPENDS_ON NONE
```

Comment:
For this and the next example below, see my comment above about whether we want to add a new DEPENDS_ON relationship type. If we don't, then these examples could be reversed:

`NONE DEPENDENCY_OF SPDXRef-CarolCompression`

Reply:
See decision from 2020/3/24 meeting.    Path of least pain is to keep NONE on RHS of expression.